### PR TITLE
Create create_wheel_wrapper.sh

### PR DESCRIPTION
**Pull request by Vipul-Ajmera**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: script/create_wheel_wrapper.sh [hunk 1]
```diff
@@ -1,61 +1,57 @@
...
         ;;
     "3.10")
         if ! python3.10 --version &>/dev/null; then
-            echo "Now installing python3.10..."
-            wget https://www.python.org/ftp/python/3.10.14/Python-3.10.14.tgz
-            tar xf Python-3.10.14.tgz
-            cd Python-3.10.14
-            echo "Building python3.10..."
+            echo "Installing dependencies required for python installation..."
+            yum install -y sudo zlib-devel wget ncurses git gcc gcc-c++ make cmake \
```
Comment (shubham-dayma-ibm) [inline script/create_wheel_wrapper.sh:26]: @nikhil-kalbande can we add complier that Amir was mentioning. If yes @Vipul-Ajmera needs to move `yum install` lines to somewhere at the top and add missing Linux distro packages from below list.
 
```
gcc
gcc-c++
gcc-gfortran
git
xz
cmake
make
yum-utils
wget
sudo
llvm
libcurl-devel
openssl
openssl-devel
libxml2-devel
bzip2-devel
libffi
libffi-devel
sqlite-devel
libpng-devel
sqlite-libs
libjpeg-devel
libicu-devel
oniguruma-devel
readline-devel
libtidy-devel
libxslt-devel
libzip-devel
diffutils
autoconf
bison-devel
git
bzip2
file
cargo
nodejs
nodejs-devel
nodejs-packaging
npm
ncurses
python3.ppc64le
python3-devel.ppc64le
zlib-devel
openblas-devel
python-clang
python3-protobuf
protobuf-c
cairo
fontconfig
yajl
tk
```

Comment (shubham-dayma-ibm) [reply script/create_wheel_wrapper.sh:26]: If we are going with above, no harm to keep below things as well.
```
# Install Rust (using rustup)
RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
ENV PATH="/root/.cargo/bin:${PATH}"

# Install Go (official binaries)
RUN wget https://golang.org/dl/go1.21.0.linux-ppc64le.tar.gz && \
    tar -C /usr/local -xzf go1.21.0.linux-ppc64le.tar.gz && \
    rm go1.21.0.linux-ppc64le.tar.gz
ENV PATH="/usr/local/go/bin:${PATH}"
```

**Review by shubham-dayma-ibm — APPROVED** [review]
LGTM, few things needs to be confirm and implemented if required

File: script/create_wheel_wrapper.sh (hunk 1 above)
Comment (nikhil-kalbande) [reply script/create_wheel_wrapper.sh:26]: For now we will not include in currency.. we want to keep installation specific to package

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**